.gitignore

```
@@ -14,3 +14,4 @@ lib/
 coverage
 packages/shared/utils
 packages/components/.out
+packages/p2p/lib/index.js
```

```suggestion
packages/p2p/lib
```
You can set it to ignore the whole `lib` folder for future proofing.